templates/articles/blocks/promoted_pages.html

```diff
@@ -29,7 +29,7 @@ <h3 class="promoted-pages__title">
                         {{ promoted_item.title }}
                     </a>
                 </h3>
-                <p class="promoted-pages__text promoted-pages__text--gap">By {{ promoted_item.author }} - {{ promoted_item.publication_date }}</p>
+                <p class="promoted-pages__text promoted-pages__text--gap">By {{ promoted_item.author }} {% if promoted_item.publication_date and promoted_item.author %}– {% endif %}{{ promoted_item.publication_date }}</p>
```

Comment:
I'm still a bit confused by this. Do we want the 'By' bit when there is no `promoted_item.author` value?

Reply:
Cheers Andy, good spot :) I've updated this now

Reply:
![image](https://github.com/nationalarchives/ds-wagtail/assets/62654785/55fe89eb-a432-4a9c-bf3f-644b52816465)

![image](https://github.com/nationalarchives/ds-wagtail/assets/62654785/badef064-0d68-46ed-9b24-e7d297acc942)

![image](https://github.com/nationalarchives/ds-wagtail/assets/62654785/30225f58-235f-4c8c-ac06-82f27db40109)

![image](https://github.com/nationalarchives/ds-wagtail/assets/62654785/bd7507ee-135e-4519-9f1b-f5c499228a53)

Reply:
Ah, yep! 
I could wrap mine in an if statement for author OR date, saves duplicating?

Reply:
This is nice and clear though:
{% if promoted_item.author %}
    <p class="promoted-pages__text promoted-pages__text--gap">
        By {{ promoted_item.author }}{% if promoted_item.publication_date %} – {{ promoted_item.publication_date }}{% endif %}
    </p>
{% elif promoted_item.publication_date %}
    <p class="promoted-pages__text promoted-pages__text--gap">
        {{ promoted_item.publication_date }}
    </p>
{% endif %}

Reply:
Cool - updated with that one :)